File: Strings/pagina03/2062/2062.cpp
#include <iostream>
using namespace std;

int main(){

    int n;
    cin >> n;

    string correct_words = "";
    for(int i = 0; i < n; i++){
        string word;
        cin >> word;

        if(!correct_words.empty()){ correct_words += " "; }

        if(word.size() == 3){
            if(word.compare(0, 2, "OB") == 0){
                correct_words += "OBI";
                continue;
            }
            if(word.compare(0, 2, "UR") == 0){
                correct_words += "URI";
                continue;
            }
        } 
            
        correct_words += word;

    }

    cout << correct_words << endl;
    return 0;
}

File: Strings/pagina03/2062/2062.js
const fs = require("fs")
const input = fs.readFileSync("/dev/stdin", "utf8").trim().split('\n')

const n_words = Number(input.shift())
const words = input.shift().split(' ')

let correct_words = []

for(let i = 0; i < n_words; i++){
    let word = words[i]
    if(word.length === 3){
        if(word.startsWith("OB")){
            correct_words.push('OBI')
        }
        else if(word.startsWith("UR")){
            correct_words.push('URI')
        }
        else {
            correct_words.push(word)
        }
    } else {
        correct_words.push(word)
    }
}

console.log(correct_words.join(' ').trim())